src/main/java/com/novoda/notils/logger/simple/Log.java

```diff
@@ -177,8 +177,8 @@ private static String getDetailedLog(String msg, int depth) {
         Thread current = Thread.currentThread();
         final StackTraceElement trace = current.getStackTrace()[depth];
         final String filename = trace.getFileName();
-        return "[" + current.getName() + "][" + filename.substring(0, filename.length() - DOT_CLASS) + "."
-                + trace.getMethodName() + ":" + trace.getLineNumber() + "] " + msg;
+        return "[" + current.getName() + "][" + "(" + filename.substring(0, filename.length() - DOT_CLASS) + ".java:"
+                + trace.getLineNumber() + ")." + trace.getMethodName() + "] " + msg;
```

Comment:
As I understand it the linking happens automatically if Name and line
number follow each other. So the method name was destroying that.
On May 21, 2015 11:40 PM, "Paul Blundell" <notifications@github.com> wrote:

> In src/main/java/com/novoda/notils/logger/simple/Log.java
> <https://github.com/novoda/notils/pull/60#discussion_r30850719>:
>
> > @@ -177,8 +177,8 @@ private static String getDetailedLog(String msg, int depth) {
> >          Thread current = Thread.currentThread();
> >          final StackTraceElement trace = current.getStackTrace()[depth];
> >          final String filename = trace.getFileName();
> > -        return "[" + current.getName() + "][" + filename.substring(0, filename.length() - DOT_CLASS) + "."
> > -                + trace.getMethodName() + ":" + trace.getLineNumber() + "] " + msg;
> > +        return "[" + current.getName() + "][" + "(" + filename.substring(0, filename.length() - DOT_CLASS) + ".java:"
> > +                + trace.getLineNumber() + ")." + trace.getMethodName() + "] " + msg;
>
> why move the line number and method name around? or is this involved in
> the linking
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/novoda/notils/pull/60/files#r30850719>.
>

-- 
Disclaimer: The information in this e-mail and any attachments is the 
property of Novoda Ltd and is confidential and may be legally privileged. 
This e-mail is intended solely for the person or organisation to which it 
is addressed. Any disclosure, copying or other use of the information by 
any person or organisation who is not the intended recipient is strictly 
prohibited and may be unlawful. If you have received this e-mail in error, 
please inform the sender immediately and delete/destroy this e-mail and any 
copies of it. Novoda Ltd has taken reasonable precautions to minimise the 
risk of any software viruses which may damage your systems, but we advise 
that you take the necessary steps to ensure that no virus contamination is 
suffered. Novoda Ltd does not accept any liability for any loss or damage 
caused by the transmission of any virus.

Novoda Ltd, Company No: 347444, Registered in Scotland Registered 
Office: C/O Alexander Sloan, 38 Cadogan Street, Glasgow, G2 7HF, Scotland.  VAT 
Registration Number  GB 984 2525 93